app/assets/javascripts/float-kit/addon/lib/d-toast-instance.js

```diff
@@ -19,4 +20,9 @@ export default class DToastInstance {
   close() {
     this.toasts.close(this);
   }
+
+  get isValidForView() {
+    const view = this.site.desktopView ? "desktop" : "mobile";
+    return this.options.views.includes(view);
```

Comment:
```suggestion
    return this.options.views.includes(this.site.desktopView ? "desktop" : "mobile");
```

Really optional, if you don't want to change it's fine :+1: